draft-irtf-cfrg-aead-limits.md

```diff
@@ -90,6 +90,15 @@ normative:
       - ins: V. T. Hoang
       - ins: S. Tessaro
       - ins: A. Thiruvengadam
+  ChaCha20Poly1305-MU:
+    title: "The Security of ChaCha20-Poly1305 in the Multi-User Setting"
+    target: https://doi.org/10.1145/3460120.3484814
+    date: 2021-11
+    author: 
+      - ins: J. P. Degabriele
+      - ins: J. Govinden
+      - ins: F. Günther
+      - ins: K. Paterson
```

Comment:
```suggestion
  ChaCha20Poly1305-MU: DOI.10.1145/3460120.3484814
```

Then we're pulling from the definitions elsewhere.  The result is the same, but we get the formatting consistent.